core-services/pr-reminder/_config.yaml

```diff
@@ -174,3 +174,4 @@ teams:
   - openshift-team-hypershift
   repos:
   - openshift/hypershift
+  channel: G01QS0P2F6W
```

Comment:
Checked 
https://github.com/slack-go/slack/blob/41fa1e5b4adad73cf664f8dedb80fb05d3e67eee/chat.go#L129
Although the arg is called channelID but its used
https://github.com/slack-go/slack/blob/41fa1e5b4adad73cf664f8dedb80fb05d3e67eee/chat.go#L266

https://api.slack.com/methods/chat.postMessage
It seems it could be both channel ID or name.
Could we use channel name for better readability? Worth giving it a shot?